.github/workflows/ci.yml

```diff
@@ -11,26 +11,29 @@ on:
 
 jobs:
   build:
-    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-20.04 ]
-        ros_distribution: [ rolling ]
         include:
-          - docker_image: ubuntu:jammy
-            ros_distribution: rolling
-    container:
-      image: ${{ matrix.docker_image }}
+          - ros_distribution: "foxy"
+            os: "ubuntu-20.04"
+          - ros_distribution: "galactic"
+            os: "ubuntu-20.04"
+          - ros_distribution: "humble"
+            os: "ubuntu-22.04"
+          - ros_distribution: "rolling"
+            os: "ubuntu-22.04"
+    runs-on: ${{ matrix.os }}
     steps:
     - uses: actions/checkout@v2
-    - uses: ros-tooling/setup-ros@0.3.0
+    - uses: ros-tooling/setup-ros@0.3.3
       with:
         use-ros2-testing: true
         required-ros-distributions: ${{ matrix.ros_distribution }}
     - name : Download and install dependencies
       run: |
-        apt-get install ros-${{ matrix.ros_distribution }}-rclc
+        sudo apt-get install ros-${{ matrix.ros_distribution }}-rclc
+        sudo apt-get install ros-${{ matrix.ros_distribution }}-osrf-testing-tools-cpp
```

Comment:
```suggestion
        apt-get install ros-${{ matrix.ros_distribution }}-rclc
        apt-get install ros-${{ matrix.ros_distribution }}-osrf-testing-tools-cpp
```

Reply:
The reason why I had to add sudo is because I got "insufficient permissions" in one of the runs (see [pipeline](https://github.com/micro-ROS/micro_ros_diagnostics/actions/runs/3513484928/jobs/5886362365#step:4:10)). I had it without sudo before